toxav/rtp.c

```diff
@@ -578,7 +578,10 @@ static int handle_rtp_packet(Messenger *m, uint32_t friendnumber, const uint8_t
         /* Store message.
          */
         session->mp = new_message(&header, header.data_length_lower, data + RTP_HEADER_SIZE, length - RTP_HEADER_SIZE);
-        memmove(session->mp->data + header.offset_lower, session->mp->data, session->mp->len);
+
+        if (session->mp != nullptr) {
```

Comment:
Is this a normal thing to happen? Should it be an error, logged, and return -1?

Reply:
Yeah I wasn't sure about that. It can only occur on a calloc failure, but perhaps it should be logged.